iChef-WebApi/src/main/java/com/kitchen/iChef/Controller/Model/Request/RecipeRequest.java

```diff
@@ -0,0 +1,51 @@
+package com.kitchen.iChef.Controller.Model.Request;
+
+import lombok.Getter;
+import lombok.Setter;
+import lombok.ToString;
+
+import javax.validation.Valid;
+import javax.validation.constraints.Min;
+import javax.validation.constraints.NotBlank;
+import javax.validation.constraints.NotEmpty;
+import java.util.List;
+
+@Getter
+@Setter
+@ToString
+public class RecipeRequest {
+    @NotBlank(message = "The title of the recipe is not valid!")
+    private String title;
+
+    @NotBlank(message = "The utensils of the recipe is not valid!")
+    private String utensils;
+
+    @NotBlank(message = "The steps of the recipe is not valid!")
+    private String steps;
+
+    @Min(value = 1, message = "The rating of the recipe is not valid!")
+    private Float rating;
+
+    @Min(value = 1, message = "The difficulty of the recipe is not valid!")
+    private Float difficulty;
+
+    @Min(value = 1, message = "The preparationTime of the recipe is not valid!")
+    private Integer preparationTime;
+
+    @Min(value = 1, message = "The portions of the recipe is not valid!")
+    private Integer portions;
+
+    @NotBlank(message = "The notes of the recipe is not valid!")
+    private String notes;
+
+    @Min(value = 1, message = "The title of the recipe is not valid!")
+    private Integer numberOfViews;
```

Comment:
Same as for the ratings, you don't want someone to create a recipe and start with 1000 views. This field should only be updated when someone visits the recipe details page.